src/renderer/index.tsx

```diff
@@ -2,9 +2,8 @@ import { createRoot } from 'react-dom/client';
 
 import 'nprogress/nprogress.css';
 import 'tailwindcss/tailwind.css';
+import { App } from './App';
 
-import { App } from './app';
-
-const container = document.getElementById('gitify');
+const container = document.getElementById('root');
```

Comment:
IMO "root" could be a bit more specific. Root what?

Reply:
it's the root div container within the body.  Seems to be commonly used in the templates I've looked at.  

If we feel its better, happy to leave the root div as `gitify`

Reply:
Yeah it's overly picky, all good :)